frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -220,8 +223,9 @@ const setup = async ({
 
   const mockEventListener = jest.spyOn(window, "addEventListener");
 
-  renderWithProviders(
+  const { history } = renderWithProviders(
     <Route>
+      <Route path="/home" component={TestHome} />
       <IndexRoute component={TestQueryBuilder} />
```

Comment:
Not from this PR, but I think this line is redundant. `IndexRoute` accepts the same path as its parent route, but it doesn't have any parent route atm. I guess this would mean this `TestQueryBuilder` would be rendered at `/` but I don't think we have a test for that.

Removing this line doesn't fail the test so I think it should be safe.

Reply:
eb51627603fef3af743752464d6309c6b0c05a47